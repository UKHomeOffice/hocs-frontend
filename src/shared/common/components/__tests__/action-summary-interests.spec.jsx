import React from 'react';
import ActionSummaryInterests from '../summary/actions/action-summary-interests';

describe('action-summary-interests.jsx', () => {

    it('should render one with details and one without', () => {

        const props = {
            items: [
                {
                    interestedPartyTitle: 'Interested party 1',
                    detailsOfInterest: 'Details of interest',
                },{
                    interestedPartyTitle: 'Interested party 1',
                },
            ]
        };

        expect(render(
            <>
                <ActionSummaryInterests props={props} />
            </>
        )).toMatchSnapshot();
    });
});