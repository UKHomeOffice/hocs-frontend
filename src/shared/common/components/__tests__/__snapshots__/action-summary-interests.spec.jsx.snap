// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`action-summary-interests.jsx should render one with details and one without 1`] = `
Array [
  <h2
    class="govuk-heading-m"
  >
    Interest
  </h2>,
  <table
    class="govuk-table margin-left--small"
  >
    <caption
      class="govuk-table__caption margin-bottom--small"
    >
      Interest 1
    </caption>
    <tbody
      class="govuk_table__body"
    >
      <tr
        class="govuk-table__row"
      >
        <th
          class="govuk-table__header padding-left--small govuk-!-width-one-third"
        >
          Party
        </th>
        <td
          class="govuk-table__cell"
        >
          Interested party 1
        </td>
      </tr>
      <tr
        class="govuk-table__row"
      >
        <th
          class="govuk-table__header padding-left--small govuk-!-width-one-third"
        >
          Details
        </th>
        <td
          class="govuk-table__cell"
        >
          Details of interest
        </td>
      </tr>
    </tbody>
  </table>,
  <table
    class="govuk-table margin-left--small"
  >
    <caption
      class="govuk-table__caption margin-bottom--small"
    >
      Interest 2
    </caption>
    <tbody
      class="govuk_table__body"
    >
      <tr
        class="govuk-table__row"
      >
        <th
          class="govuk-table__header padding-left--small govuk-!-width-one-third"
        >
          Party
        </th>
        <td
          class="govuk-table__cell"
        >
          Interested party 1
        </td>
      </tr>
    </tbody>
  </table>,
]
`;
